d2l-table-shared-styles.js

```diff
@@ -50,7 +50,7 @@ $_documentContainer.innerHTML = `<custom-style>
 				background-color:var(--d2l-table-light-header-background-color);
 				color: var(--d2l-color-ferrite);
 				font-size: 0.7rem;
-				font-weight: regular
+				font-weight: normal;
```

Comment:
Haha I guess `regular` isn't a thing eh

Reply:
Yeah lol, you shouldn't copy/paste right from the design specs.  It _seemed_ familiar though, like almost a real thing!  Edge was the only browser who didn't gracefully ignore